docs/integrations/databases/mongodb-atlas.md

```diff
@@ -371,65 +371,66 @@ MongoDBAtlas:
-        - MEMORY_RESIDENT
-        - MEMORY_VIRTUAL
-        - MEMORY_MAPPED
-        - NETWORK_BYTES_IN
-        - NETWORK_BYTES_OUT
-        - NETWORK_NUM_REQUESTS
-        - OPCOUNTER_CMD
-        - OPCOUNTER_QUERY
-        - OPCOUNTER_UPDATE
-        - OPCOUNTER_DELETE
-        - OPCOUNTER_GETMORE
-        - OPCOUNTER_INSERT
-        - OP_EXECUTION_TIME_READS
-        - OP_EXECUTION_TIME_WRITES
-        - OP_EXECUTION_TIME_COMMANDS
-        - OPLOG_MASTER_LAG_TIME_DIFF
-        - OPLOG_SLAVE_LAG_MASTER_TIME
-        - QUERY_EXECUTOR_SCANNED
-        - QUERY_EXECUTOR_SCANNED_OBJECTS
-        - QUERY_TARGETING_SCANNED_PER_RETURNED
-        - QUERY_TARGETING_SCANNED_OBJECTS_PER_RETURNED
-        - SYSTEM_NORMALIZED_CPU_USER
-        - SYSTEM_NORMALIZED_CPU_KERNEL
-        - SYSTEM_NORMALIZED_CPU_IOWAIT
-        - PROCESS_CPU_USER
-        - PROCESS_CPU_KERNEL
-        - SYSTEM_NORMALIZED_CPU_STEAL
-
-      DISK_METRICS:
-        - DISK_PARTITION_IOPS_READ
-        - DISK_PARTITION_IOPS_WRITE
-        - DISK_PARTITION_UTILIZATION
-        - DISK_PARTITION_LATENCY_READ
-        - DISK_PARTITION_LATENCY_WRITE
-        - DISK_PARTITION_SPACE_PERCENT_FREE
-        - DISK_PARTITION_SPACE_PERCENT_USED
+  LOG_TYPES:
+    - DATABASE
+    - AUDIT
+    - EVENTS_PROJECT
+    - EVENTS_ORG
+    - ALERTS
+
+  METRIC_TYPES:
+    PROCESS_METRICS:
+      - CACHE_DIRTY_BYTES
+      - CACHE_USED_BYTES
+      - CONNECTIONS
+      - CURSORS_TOTAL_OPEN
+      - CURSORS_TOTAL_TIMED_OUT
+      - DATABASE_AVERAGE_OBJECT_SIZE
+      - DB_STORAGE_TOTAL
+      - DB_DATA_SIZE_TOTAL
+      - EXTRA_INFO_PAGE_FAULTS
+      - GLOBAL_LOCK_CURRENT_QUEUE_TOTAL
+      - MEMORY_RESIDENT
+      - MEMORY_VIRTUAL
+      - MEMORY_MAPPED
+      - NETWORK_BYTES_IN
+      - NETWORK_BYTES_OUT
+      - NETWORK_NUM_REQUESTS
+      - OPCOUNTER_CMD
+      - OPCOUNTER_QUERY
+      - OPCOUNTER_UPDATE
+      - OPCOUNTER_DELETE
+      - OPCOUNTER_GETMORE
+      - OPCOUNTER_INSERT
+      - OP_EXECUTION_TIME_READS
+      - OP_EXECUTION_TIME_WRITES
+      - OP_EXECUTION_TIME_COMMANDS
+      - OPLOG_MASTER_LAG_TIME_DIFF
+      - OPLOG_SLAVE_LAG_MASTER_TIME
+      - QUERY_EXECUTOR_SCANNED
+      - QUERY_EXECUTOR_SCANNED_OBJECTS
+      - QUERY_TARGETING_SCANNED_PER_RETURNED
+      - QUERY_TARGETING_SCANNED_OBJECTS_PER_RETURNED
+      - SYSTEM_NORMALIZED_CPU_USER
+      - SYSTEM_NORMALIZED_CPU_KERNEL
+      - SYSTEM_NORMALIZED_CPU_IOWAIT
+      - PROCESS_CPU_USER
+      - PROCESS_CPU_KERNEL
+      - SYSTEM_NORMALIZED_CPU_STEAL
+
+    DISK_METRICS:
+      - DISK_PARTITION_IOPS_READ
+      - DISK_PARTITION_IOPS_WRITE
+      - DISK_PARTITION_UTILIZATION
+      - DISK_PARTITION_LATENCY_READ
+      - DISK_PARTITION_LATENCY_WRITE
+      - DISK_PARTITION_SPACE_PERCENT_FREE
+      - DISK_PARTITION_SPACE_PERCENT_USED
 
 Collection:
   DBNAME: "newmongodbatlas"
 ```
+Example execution of second yaml file: /usr/bin/python3 -m sumomongodbatlascollector.main <path-of-second-yaml-file>
```

Comment:
```suggestion
Example execution of second yaml file: `/usr/bin/python3 -m sumomongodbatlascollector.main <path-of-second-yaml-file>`
```

Reply:
I had to place the command into a code block before it would compile using yarn.